Connectors/Tekla/Speckle.Connector.TeklaShared/HostApp/SendCollectionManager.cs

```diff
@@ -0,0 +1,58 @@
+using Speckle.Converter.Tekla2024;
+using Speckle.Converters.Common;
+using Speckle.Sdk.Models.Collections;
+
+namespace Speckle.Connector.Tekla2024.HostApp;
```

Comment:
Namespace need to be update to shared one

Reply:
Problem across most `.cs` files. @dogukankaratas will clean this up all affected files :)

Reply:
I see. @dogukankaratas (if no any already) could you also open a ticket for this trivial task to not forget?